setup.py

```diff
@@ -121,6 +95,20 @@ def get_include_dirs():
 
     return include_dirs
 
+# TODO Do we need to determine package data dynamically? If so, should
```

Comment:
ideally we would determine dynamically, but I think that could be left to a future PR if it ends up being needed.  One example is the proposed CMAKE-based Windows FFTW build in conda-forge/fftw-feedstock#32 which does not build the long-double variant and the library names end up being just `fftw3.dll` and `fftw3f.dll`.